data/metadata_test/_validation_schema.toml

```diff
@@ -97,7 +98,7 @@ type = "string"
 
 [[environment.columns]]
 name = "host_id"
-type = "string"
```

Comment:
Are you sure we need to specify the size of the integer? Perhaps int should be enough?

Reply:
That is how it comes out of pandas ... I do not like it that way. For now I will leave it in and we can refine from here.